cpp/include/cudf/functions.h

```diff
@@ -845,3 +845,6 @@ gdf_error gdf_from_dlpack(gdf_column** columns,
 gdf_error gdf_to_dlpack(DLManagedTensor_ *tensor,
                         gdf_column const * const * columns,
                         gdf_size_type num_columns);
+
+
+bit_mask::bit_mask_t* nans_to_nulls(gdf_column const* col);
```

Comment:
Inputs are pass by const ref.
```suggestion
bit_mask::bit_mask_t* nans_to_nulls(gdf_column const& col);
```